robotoff/prediction/ocr/core.py

```diff
@@ -55,54 +65,61 @@ def get_json_for_image(barcode: str, image_id: str) -> Optional[JSONType]:
     return r.json()
 
 
-def extract_predictions(
-    content: Union[OCRResult, str], prediction_type: PredictionType
-) -> List[Prediction]:
-    """Proxy to each predictor, depending on prediction type."""
-    if prediction_type == PredictionType.packager_code:
-        return find_packager_codes(content)
-
-    elif prediction_type == PredictionType.label:
-        return find_labels(content)
-
-    elif prediction_type == PredictionType.expiration_date:
-        return find_expiration_date(content)
-
-    elif prediction_type == PredictionType.image_flag:
-        return flag_image(content)
-
-    elif prediction_type == PredictionType.image_orientation:
-        return find_image_orientation(content)
+def get_ocr_result(
+    ocr_url: str, session: requests.Session, error_raise: bool = True
+) -> Optional[OCRResult]:
```

Comment:
should'nt we move this to off.py ?

Reply:
It's not really off-dependent and is tightly linked to OCR JSON parsing so I would better keep it here.